api/v1beta1/openstackdataplanenodeset_webhook.go

```diff
@@ -126,16 +127,25 @@ func (r *OpenStackDataPlaneNodeSet) ValidateCreate() (admission.Warnings, error)
 	if err != nil {
 		return nil, err
 	}
+	// Check if OpenStackDataPlaneNodeSet name matches RFC1123 for use in labels
+	validate := validator.New()
+	if err = validate.Var(r.Name, "hostname_rfc1123"); err != nil {
+		openstackdataplanenodesetlog.Error(err, "Error validating OpenStackDataPlaneNodeSet name, name must follow RFC1123")
+		errors = append(errors, field.Invalid(
+			field.NewPath("Name"),
```

Comment:
In general you want to return the json path of the CRD field that failed the validation. I think in this case that is field.NewPath("metadata").Child("name").